common/postgres-init.sql

```diff
@@ -1,3 +1,4 @@
 CREATE DATABASE catalog_db;
+\c catalog_db
 CREATE ROLE dbuser WITH LOGIN ENCRYPTED PASSWORD 'dbpwd';
 CREATE EXTENSION pgcrypto;
```

Comment:
```suggestion
CREATE ROLE dbuser WITH LOGIN ENCRYPTED PASSWORD 'dbpwd';
CREATE DATABASE catalog_db OWNER dbuser;
\c catalog_db
CREATE EXTENSION pgcrypto;
```

While we're fixing stuff in `postgres-init.sql`, I'm curious if instead we might want the `dbuser` to own the `catalog_db`?

Reply:
That makes sense to me!